stdlib/LinearAlgebra/test/dense.jl

```diff
@@ -39,6 +39,19 @@ Random.seed!(1234321)
         M = [1.0 -2.0; -2.0 -1.5]
         @test cond(M, 1) ≈ 2.227272727272727
     end
+    @testset "Empty matrices" begin
+        @test cond(zeros(Int, 0, 0), 1) === 0.0
+        @test cond(zeros(Int, 0, 0), 2) === 0.0
+        @test cond(zeros(Int, 0, 0), Inf) === 0.0
+        @test cond(zeros(0, 0), 1) === 0.0
+        @test cond(zeros(0, 0), 2) === 0.0
+        @test cond(zeros(0, 0), Inf) === 0.0
+        @test cond(zeros(ComplexF64, 0, 0), 1) === 0.0
+        @test cond(zeros(ComplexF64, 0, 0), 2) === 0.0
+        @test cond(zeros(ComplexF64, 0, 0), Inf) === 0.0
```

Comment:
test case for BigFloat ?

Reply:
```julia
julia> cond(ones(BigFloat, 1, 1), 2)
ERROR: MethodError: no method matching svdvals!(::Matrix{BigFloat})
```
So testing this works for `ones(BigFloat, 0, 0)` seems rather pointless.

Reply:
That differs from my experience:

```
julia> cond(ones(BigFloat, 1, 1), 2)
1.0

julia> versionifo()
ERROR: UndefVarError: versionifo not defined
Stacktrace:
 [1] top-level scope
   @ REPL[147]:1

julia> versioninfo()
Julia Version 1.6.0-DEV.1449
Commit 489808a369 (2020-11-09 06:56 UTC)
Platform Info:
  OS: Linux (x86_64-redhat-linux)
```

Reply:
https://github.com/JuliaLang/julia/blob/19506f75ae23afb7376c5a6a468e8f2b955ec48c/stdlib/LinearAlgebra/src/svd.jl#L468

Reply:
Sorry, I was mislead by this `GenericLinearAlgebra` package, which defines `svdvals!`:
```
In #svdvals!#37(tol, debug, , A) at ~/.julia/packages/GenericLinearAlgebra/JoKjw/src/svd.jl:486
 486  function LinearAlgebra.svdvals!(A::StridedMatrix; tol = eps(real(eltype(A))), debug = false)
>487      B = bidiagonalize!(A).bidiagonal
 488      # It doesn't matter that we transpose the bidiagonal matrix when we are only interested in the values
 489      return svdvals!(Bidiagonal(B.dv, B.ev, :U), tol = tol, debug = debug)
 490  end
```